pyproject.toml

```diff
@@ -59,6 +59,7 @@ Changelog = "https://github.com/ott-jax/ott/releases"
 neural = [
     "flax>=0.6.6",
     "optax>=0.1.1",
+    "diffrax>=0.4.1",
```

Comment:
Please also add explicitly `orbax-checkpoint`.

Reply:
I now removed the checkpoint manager, so won't need it anymore.